fuzzers/weizz_qemu/fuzzer.py

```diff
@@ -42,6 +42,7 @@ def fuzz(input_corpus, output_corpus, target_binary):
     os.environ['WEIZZ_I_DONT_CARE_ABOUT_MISSING_CRASHES'] = '1'
     os.environ['WEIZZ_CTX_SENSITIVE'] = '1'
     os.environ['WEIZZ_SKIP_CRASHES'] = '1'
+    os.environ['AFL_SHUFFLE_QUEUE'] = '1'  # not sure this is supported
```

Comment:
Does it use AFL_ prefix or WEIZZ_ ?

Reply:
Yes it is WEIZZ_SHUFFLE_QUEUE  
can you make the change
https://github.com/andreafioraldi/weizz-fuzzer/blob/1a2121029c0ed23e1529830dc5f4d9acabdebb10/src/main.c#L556

Reply:
had to check the repo. it actually supports it, but only with WEIZZ_ ... changed that